modules/apps/27-interchain-accounts/controller/keeper/account.go

```diff
@@ -15,8 +15,8 @@ import (
 // call 04-channel 'ChanOpenInit'. An error is returned if the port identifier is
 // already in use. Gaining access to interchain accounts whose channels have closed
 // cannot be done with this function. A regular MsgChanOpenInit must be used.
-func (k Keeper) InitInterchainAccount(ctx sdk.Context, connectionID, counterpartyConnectionID, owner string) error {
-	portID, err := icatypes.GeneratePortID(owner, connectionID, counterpartyConnectionID)
+func (k Keeper) InitInterchainAccount(ctx sdk.Context, connectionID, owner string) error {
```

Comment:
Removed `counterpartyConnectionID` as it can be retrieved using the channelKeeper below. This reduces the burden on developers implementing their own authentication modules

Reply:
Nice change. We need to update the spec should we change this @AdityaSripal @crodriguezvega 

Likewise one of us should update the ICA repo + hub teams PR (the CLI commands take two connections etc)

Reply:
We also need to update the spec to explain the json-encoded format of the version string, right?